### PR TITLE
feat(Upgrade): Add VS Code upgrade route

### PR DESCRIPTION
**Pull request by DavidHollins6**
## What kind of change does this PR introduce?
Similar to the docker sidebar button. This is a new navigation button for VS Code which encourages the user to upgrade their sandbox to V2.

## What steps did you take to test this? This is required before we can merge, make sure to test the flow you've updated.

1. Step A
2. Step B
3. Step C

## Checklist

- [ ] Documentation
- [ ] Testing 
- [ ] Ready to be merged
      
- [ ] Added myself to contributors table

### CODE REVIEW

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          To use VS Code together with CodeSandbox, you need to convert your
+          Browser Sandbox into a Cloud Sandbox.
+        </Text>
+        <Text as="p" variant="muted" margin={0}>
+          Cloud Sandboxes are an improved coding experience that runs your code
+          in the cloud. They allow you to run Docker, code in new languages, add
+          servers, databases, and much more.
+        </Text>
+        <Text as="p" variant="muted" margin={0}>
+          Don&apos;t worry-this sandbox will not be deleted. After converting,
```
Comment (olarclara) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx:27]:
```suggestion
          Don&apos;t worry&mdash;this sandbox will not be deleted. After converting,
```

**Review by olarclara — APPROVED** [review]

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx (hunk 1 above)
Comment (tristandubbeld) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx:27]: Why are we actually using a dash here?

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  Collapsible,
+  Text,
+  Button,
+  Stack,
+  Link,
+} from '@codesandbox/components';
+import { useUpgradeFromV1ToV2 } from 'app/hooks/useUpgradeFromV1ToV2';
+
+export const VSCode = () => {
+  const { perform, loading, canConvert } = useUpgradeFromV1ToV2('Docker menu');
```
Comment (tristandubbeld) [inline packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx:12]:
```suggestion
  const { perform, loading, canConvert } = useUpgradeFromV1ToV2('VSCode menu');
```

**Review by tristandubbeld — COMMENTED** [review]
Only the tracking text needs a change I think!

File: packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx (hunk 1 above)
Comment (olarclara) [reply packages/app/src/app/pages/Sandbox/Editor/Workspace/screens/VSCode/index.tsx]: @tristandubbeld the copy is a double dash, so using the char reference is the easiest way to make sure it renders correctly.

**Review by filipelima18 — APPROVED** [review]

**Pull request merged**